### PR TITLE
Feature/models

### PR DESCRIPTION
**Pull request by vasyltanchynets**

### CODE REVIEW

File: Galagram/Models/Entities/Comment.cs [hunk 1]
```diff
@@ -1,12 +1,103 @@
...
+        // CONSTRUCTORS
+        /// <summary>
+        /// Constructor with parameters
+        /// </summary>
+        /// <param name="id">Id comment</param>
+        /// <param name="author">The author who commented</param>
+        /// <param name="likes">Amount of likes</param>
+        /// <param name="text">Text of the comment</param>
+        /// <param name="date">Time to write a comment</param>
+        public Comment(int id, User author, System.Collections.Generic.ICollection<Like> likes, string text, System.DateTime date)
```
Comment (SvyatoslavFedynyak) [inline Galagram/Models/Entities/Comment.cs]: Don't need here constructors with params. Only parameterless

File: Galagram/Models/Entities/Comment.cs [hunk 2]
```diff
@@ -1,12 +1,103 @@
...
+                return date;
+            }
+            set
+            {
+                date = value;
+            }
+        }
+        // CONSTRUCTORS
+        /// <summary>
+        /// Constructor with parameters
```
Comment (SvyatoslavFedynyak) [inline Galagram/Models/Entities/Comment.cs]: Don't need here constructors with params. Only parameterless. In all entities, so it should be 6 changes

File: Galagram/Models/Entities/Message.cs [hunk 3]
```diff
@@ -1,12 +1,103 @@
...
     public class Message
     {
-        // Properties
-        public int Id { get; set; }
-        public string Text { get; set; }
-        public User User { get; set; }
-        public Subject Subject { get; set; }
-        public System.DateTime Date { get; set; }
+        // FIELDS
+        private int id;
```
Comment (SvyatoslavFedynyak) [inline Galagram/Models/Entities/Message.cs:9]: Not important. Fields are private by default.

**Review by SvyatoslavFedynyak — CHANGES_REQUESTED** [review]

File: Galagram/Models/Entities/Message.cs (hunk 3 above)
Comment (iamprovidence) [reply Galagram/Models/Entities/Message.cs:9]: however it not a problem. and code become easier to read

Comment (SvyatoslavFedynyak) [reply Galagram/Models/Entities/Message.cs:9]: > 
> 
> however it not a problem. and code become easier to read
>
I don't think its easier, and if you'll spend 5 sec for every "private" you may lose a minute for every 12 fields)

Comment (iamprovidence) [reply Galagram/Models/Entities/Message.cs:9]: still think it is not a problem. you know, sometimes missing access modifier can lead to more problems than missing minute, for example missing an hour to rework project carcass. but as you think

Comment (SvyatoslavFedynyak) [reply Galagram/Models/Entities/Message.cs:9]: I don't want him to rework it. Only for future

Comment (iamprovidence) [reply Galagram/Models/Entities/Message.cs:9]: Even for future, still think it is not a problem. But as you wish

File: Galagram/Models/Entities/Comment.cs (hunk 2 above)
Comment (vasyltanchynets) [reply Galagram/Models/Entities/Comment.cs]: then constructor with parameters at all to remove ?

Comment (SvyatoslavFedynyak) [reply Galagram/Models/Entities/Comment.cs]: Yes. Need only default

Comment (SvyatoslavFedynyak) [reply Galagram/Models/Entities/Comment.cs]: Yes. We need just default.

**Review by SvyatoslavFedynyak — APPROVED** [review]

**Pull request merged**